### PR TITLE
MULTIARCH-2678: Incorporating Eltorito Boot loader fix in agent 

### PR DESCRIPTION
**Pull request by phani0041gs**
Incorporating Eltorito Bootloader fix in agent installer from assisted-image-service for s390x

Reference PR : https://github.com/openshift/assisted-image-service/pull/164

Bug Fix : https://issues.redhat.com/browse/MGMT-16264

The above bug is fixed in the referred PR, to incorporate this in the agent-based-installer updated the required module versions.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/installer
 
-go 1.20
+go 1.21
```
Comment (andfasano) [inline go.mod:3]: Is this required by some specific dependency?

File: go.mod [hunk 2]
```diff
@@ -213,9 +219,17 @@ require (
 	github.com/pkg/xattr v0.4.9 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/prometheus/client_model v0.5.0 // indirect
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (phani2898) [reply go.mod]: resolved all the conflicts now this should not be there

File: go.mod (hunk 1 above)
Comment (phani2898) [reply go.mod]: Just pulled from master and now it is synced

**Comment by andfasano** [comment]
It looks like there are some conflicts still be resolved?

Can you please also run a

```
go mod tidy
go mod vendor
```
for vendoring the required fix?

**Comment by phani0041gs** [comment]
Closing this PR based on the conversation here : https://redhat-internal.slack.com/archives/C06A57N4WEM/p1706508696035429.

Raised a new PR which supports this module update : https://github.com/openshift/installer/pull/8069

**Pull request closed without merging**